java/arcs/android/storage/ttl/PeriodicCleanupTask.kt

```diff
@@ -0,0 +1,29 @@
+package arcs.android.storage.ttl
+
+import android.content.Context
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import arcs.core.util.TaggedLog
+
+class PeriodicCleanupTask(
+    appContext: Context,
+    workerParams: WorkerParameters
+) : Worker(appContext, workerParams) {
+
+    private val log = TaggedLog { this.toString() }
+    init {
+        log.debug { "Created." }
+    }
```

Comment:
```kotlin
    private val log = TaggedLog { "PeriodCleanupTask" }
    
    init { log.debug { "Created." } }
```

Reply:
a sad story how copy&paste could fail you, if you copy from the wrong place. fixed.